.github/workflows/test.yml

```diff
@@ -41,7 +41,10 @@ jobs:
         pip install .[ml]
     - name: Test with pytest
       run: |
-        python -m pytest --random-order --random-order-bucket=global --ignore=docs -m "not api_validation" tests/
+        # unit-tests
+        python -m pytest --random-order --random-order-bucket=global --ignore=docs --ignore=tests/test_app.py tests/
+        # application test
+        python -m pytest --random-order --random-order-bucket=global --ignore=docs tests/test_app.py
```

Comment:
I assume that new test command can be separated to a new step in the `test.yml`
To add clearer divide between unit and integration tests, if we want to 

Btw, I don't see direct benefits dividing tests into 2 separate commands, they are still both just a pytest tests anyway